src/OrchardCore/OrchardCore.ResourceManagement/TagHelpers/MetaTagHelper.cs

```diff
@@ -24,6 +24,8 @@ public class MetaTagHelper : TagHelper
 
         public string Separator { get; set; }
 
+        public bool Replace { get; set; }
```

Comment:
Append

Reply:
@netwavebe can you please make these changes?